packages/pwa-buildpack/lib/WebpackTools/plugins/__tests__/UpwardDevServerPlugin.spec.js

```diff
@@ -2,14 +2,16 @@ jest.mock('node-fetch');
 jest.mock('@magento/upward-js');
 const upward = require('@magento/upward-js');
 const fetch = require('node-fetch');
-const UpwardPlugin = require('../UpwardPlugin');
+const UpwardDevServerPlugin = require('../UpwardDevServerPlugin');
+
+const mockContext = require('path').resolve(__dirname, '../../../../../../');
```

Comment:
This is the path to our project root I'm guessing? If so, may want to drop a comment or rename the variable `projectRoot` or something

Reply:
A comment would suffice, I think.

Reply:
Comment added.